projects/knora-ui/src/lib/viewer/values/boolean-value/boolean-value.component.html

```diff
@@ -1,19 +1,25 @@
-<span [formGroup]="form">
-    <mat-checkbox [formControlName]="'booleanValue'" class="value boolValue child-value-component" (change)="onChecked()">
-        {{booleanLabel}}
-    </mat-checkbox>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-        </textarea>
-    </mat-form-field>
-</span>
\ No newline at end of file
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p class="rm-value">{{booleanLabel}}</p>
+    <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+        <mat-checkbox [formControlName]="'booleanValue'" class="value boolValue child-value-component" (change)="onChecked()">
```

Comment:
The class contains some logic for disabling the checkbox / `FormControl` when in read mode:

```ts
    // override the resetFormControl() from the base component to deal with the disabled state and the checkbox label
    resetFormControl(): void {
        super.resetFormControl();

        if (this.valueFormControl !== undefined) {
            this.setBoolLabel();
            if (this.mode === 'read') {
                this.valueFormControl.disable();
            } else {
                this.valueFormControl.enable();
            }
        }

    }
```

I think some of this logic is not needed anymore.

Reply:
to discuss with Flavie

Reply:
to be done as part of https://dasch.myjetbrains.com/youtrack/issue/DSP-206